CONTRIBUTING.de.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+:crossed_flags:
+**Deutsch**,
+[English][EN],
+[Español][ES],
+[Français][FR],
+[Nederlands][NL],
+[Português][PT_BR],
+[Türkçe][TR],
+[العربية][AR],
+[正體中文][ZH_TW],
+[简体中文][ZH_CN],
+[한국어][KO_KR]
+
+[AR]:CONTRIBUTING.ar.md
+[DE]:CONTRIBUTING.de.md
+[EN]:CONTRIBUTING.md
+[ES]:CONTRIBUTING.es.md
+[FR]:CONTRIBUTING.fr.md
+[KO_KR]:CONTRIBUTING.ko_kr.md
+[NL]:CONTRIBUTING.nl.md
+[PT_BR]:CONTRIBUTING.pt_br.md
+[TR]:CONTRIBUTING.tr.md
+[ZH_CN]:CONTRIBUTING.zh_cn.md
+[ZH_TW]:CONTRIBUTING.zh_tw.md
+
+Der Quellcode in diesem repository wurde manuell digitalisiert, also sind jegliche schriftliche Fehler und Ungereimtheiten aus versehen hinzugefügt worden. Der Code soll Modifiziert werden so dass er mit den Vorlagen übereinstimmt. 
+
+* [AGC printouts for Comanche][8]
+* [AGC printouts for Luminary][9]
+
+## Nützliche Erweiterungen
+
+GitHub hat eine Integrierte Unterstützung für AGC assembly. Dasselbe gilt für ihren Code Editor nicht, dennoch gibt es AGC Erweiterungen die Syntax highlighting für die folgenden Editors zur Verfügung stellen: 
+- [Atom][Atom]†
+- [CodeBlocks][CodeBlocks]
+- [Eclipse][Eclipse]
+- [Kate][Kate]
+- [ProgrammersNotepad][ProgrammersNotepad]
+- [Sublime Text 3][Sublime Text]†
+- [TextPad][TextPad]
+- [Vim][Vim]
+- [Visual Studio Code][VisualStudioCode]†
+- [jEdit][jEdit]
+
+† Unterstützen Automatisches Formatieren  
+
+[Atom]:https://github.com/Alhadis/language-agc
+[CodeBlocks]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/CodeBlocks
+[Eclipse]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/Eclipse
+[Kate]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/Kate
+[ProgrammersNotepad]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/ProgrammersNotepad
+[Sublime Text]:https://github.com/jimlawton/AGC-Assembly
+[TextPad]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/TextPad
+[Vim]:https://github.com/wsdjeg/vim-assembly
+[VisualStudioCode]:https://github.com/wopian/agc-assembly
+[jEdit]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/jEdit
+
+## Formatierung  
+**Anmerkung:** Die oben erwähnten GitHub Erweiterungen werden automatisch sicherstellen dass sie die Korrekte Formatierung einhalten.
```

Comment:
Is this "GitHub extensions mentioned above"? If it is, it should be "GitHub **and** [the] extensions mentioned above.

Just basing this of Google Translate as I don't know German.